pkg/storage/utils/decomposedfs/decomposedfs.go

```diff
@@ -655,7 +656,10 @@ func (fs *Decomposedfs) TouchFile(ctx context.Context, ref *provider.Reference,
 	if err := n.CheckLock(ctx); err != nil {
 		return err
 	}
-	return fs.tp.TouchFile(ctx, n, markprocessing)
+	if err = n.SetXattrString(prefixes.MetadataPrefix+mtime, mtime); err != nil {
+		return err
+	}
```

Comment:
This should be done by the `tree` just like setting `markprocessing`. 

See here: https://github.com/cs3org/reva/pull/4012/files#diff-f46b4dbde0970bc0eeea2d36b8966b3fb084404cf52a006a12da0050fb8db936R156